packages/key-management/test/Bip32Account.test.ts

```diff
@@ -121,4 +146,20 @@ describe('Bip32Account', () => {
       );
     });
   });
+
+  describe('derives CIP-1854 address', () => {
```

Comment:
How is this test asserting CIP-1854? :thinking: There are no changes in Bip32Account, so we probably don't need to have any additional tests for it